lib/components/chat_history/auxiliary/realtimecash_donation.dart

```diff
@@ -17,11 +17,13 @@ class RealtimeCashDonationEventWidget extends StatelessWidget {
         Text.rich(
           TextSpan(
             children: [
-              const TextSpan(text: "You received "),
+              TextSpan(text: "${model.donor} donated "),
               TextSpan(
-                  text: "${model.value.toString()} ${model.currency}.",
+                  text: "${model.value.toString()} ${model.currency}. ",
                   style: boldStyle),
-              const TextSpan(text: " Your transaction hash: "),
+              if (model.message.isNotEmpty)
+                TextSpan(text: "message: ${model.message}"),
```

Comment:
Render the message in italics below the donation notification message: https://github.com/muxable/rtchat/blob/c16ceff81d06e99a32f400174911c48cda6064a8/lib/components/chat_history/auxiliary/streamlabs.dart#L27-L29